### PR TITLE
Remove old feature flag

### PR DESCRIPTION
**Pull request by nigeldunne**
### Change description ###

Remove old feature flag

### JIRA link (if applicable) ###

N/A

### Pull request checklist ###

**Before raising**
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] README and other documentation has been updated / added (if needed)

**Before merging**
- [ ] this ticket been reviewed by QA
- [ ] the user story been signed off by the PO

**Note:** Bug fixes, dependency updates and technical tasks do not directly impact the user experience and can be merged without QA and PO review.

### CODE REVIEW

File: src/main/resources/application.yaml [hunk 1]
```diff
@@ -569,5 +569,6 @@ bulk-action:
 
 SERVICE_AUTH_MICROSERVICE: nfdiv_case_api
 
-toggle:
-  enable_qr_code_reading: ${ENABLE_QR_CODE_READING:true}
+logging:
```
Comment (linusnorton) [inline src/main/resources/application.yaml:572]: Is there a reason you added this?

Comment (nigeldunne) [reply src/main/resources/application.yaml:572]: I need to add this when I'm running locally to so that I can see the log messages

Comment (linusnorton) [reply src/main/resources/application.yaml:572]: what impact will it have on production?

Comment (nigeldunne) [reply src/main/resources/application.yaml]: I don't how how logging it configred in production, I have removed this for now.

**Pull request merged**